participant/stories/components/TextField.stories.tsx

```diff
@@ -64,6 +64,19 @@ export const TextInputLargeStyleRequired = {
   },
 };
 
+export const TextInputLargeStyleWithHint = {
+  ...TextFieldTemplate,
+  args: {
+    ...defaultProps,
+    labelClassName: "usa-label--large",
+    hint: (
+      <div>
+        <em>Helper text</em>
+      </div>
```

Comment:
To more closely match the actual content, this should not be wrapped in a `<div>` tag.